### PR TITLE
change cos link

### PR DESCRIPTION
**Pull request by bdwyer2**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,6 +1,6 @@
 FROM codait/max-base:v1.1.3
 
-ARG model_bucket=http://max-assets.s3.us.cloud-object-storage.appdomain.cloud/image-segmenter/1.0
+ARG model_bucket=https://max-assets-dev.s3.us-south.cloud-object-storage.appdomain.cloud/max-image-segmenter/1.0.0
```
Comment (kmh4321) [inline Dockerfile]: can you change it to `max-assets-prod`? I have already uploaded the assets on prod for the other deployment.

Comment (xuhdev) [reply Dockerfile]: And is `-south` necessary? I thought we had some discussion on slack...

Comment (kmh4321) [reply Dockerfile]: One of them was the old URL. We can't/shouldn't use that anymore.

**Comment by djalova** [comment]
Can you also rename the assets folder?

**Comment by kmh4321** [comment]
Actually I also made changes to the file names when uploading to `prod` as part of the housekeeping changes (which are unfortunately clubbed with the training scripts) so travis miight fail. If it does, I'll fix it.

**Pull request merged**